nightwatch/tests/loginVCAViewPatientData.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
Some files use `"use strict";` and others user `'use strict';`.  Please standardize on the latter.